ui/store.js

```diff
@@ -96,7 +96,16 @@ const persistOptions = {
 
 let history;
 // @if TARGET='app'
-history = createHashHistory();
+let initialEntry = '/';
+let hash = window.location.hash;
+if (hash) {
+  hash = hash.replace('#', '');
+  initialEntry = hash.startsWith('/') ? hash : '/' + hash;
+}
+history = createMemoryHistory({
+  initialEntries: [initialEntry],
```

Comment:
@seanyesmunt @tzarebczan After way too much jiggytom I found the root of the issue. It seems the default initialEntries were overriding the entry sent over by lbry.tv.

Reply:
@dalhill This fixes opening `lbry://xxx` links?

Reply:
Yes, links like the following.

![image](https://user-images.githubusercontent.com/15717854/74076857-caac1a80-49e0-11ea-8ceb-1ba84fb44792.png)

I've tested this on videos, channels, "settings" and the main page. The only weird one so far is the main page as this makes the app open up into the "discover" view, however, this doesn't seem to be a bug related to the code in this PR. 

If you know of any "types" of links that I've missed, let me know and I'll test those as well.

Reply:
Awesome!!!

The only other thing I would test is if this contains the correct path:
https://github.com/lbryio/lbry-desktop/blob/master/ui/analytics.js#L250

It's the first initial page view event we send. You could just console log the url there.

Reply:
Yea this initial event was sending "/" instead of "/jiggytom". I pushed a change that fixes this.